backend/controllers/projects.go

```
@@ -1439,3 +1453,90 @@ func AutomergePRforBatchIfEnabled(gh utils.GithubClientProvider, batch *models.D
+
+	deleteOlderComments := config.DeletePriorComments
+
+	slog.Debug("Delete prior comments settings",
+		"enabled", deleteOlderComments,
+		"batchStatus", batch.Status,
+		"batchType", batch.BatchType,
+	)
+
+	if (batch.Status == orchestrator_scheduler.BatchJobSucceeded || batch.Status == orchestrator_scheduler.BatchJobFailed) &&
+		batch.BatchType == orchestrator_scheduler.DiggerCommandPlan &&
+		batch.CoverAllImpactedProjects == true &&
```

golangci-lint (1.64.8)

S1002: should omit comparison to bool constant, can be simplified to `batch.CoverAllImpactedProjects`

(gosimple)